src/tests/cmocka/data_provider/test_dp_request.c

```diff
@@ -89,6 +89,8 @@ bool __wrap_be_is_offline(struct be_ctx *ctx)
 #define NAME      "test_user"
 #define NAME2     "test_user2"
 #define REQ_NAME  "getpwuid"
+#define CID  "1"
```

Comment:
Hi,

the 5th argument of `dp_req_send()` is an `uint32_t` not a string.

bye,
Sumit

Reply:
Fixed.